src/khoj/configure.py

```diff
@@ -80,46 +68,56 @@ def configure_server(config: FullConfig, regenerate: bool, search_type: Optional
     finally:
         state.config_lock.release()
 
+
+def initialize_content(regenerate: bool, search_type: Optional[SearchType] = None) -> None:
     # Initialize Content from Config
     if state.search_models:
         try:
-            state.config_lock.acquire()
-            state.content_index = configure_content(
-                state.content_index, state.config.content_type, state.search_models, regenerate, search_type
-            )
+            logger.info("📬 Updating content index Scheduler")
+            all_files = collect_files(state.config.content_type)
+            url = f"http://0.0.0.0:42110/indexer/batch?regenerate={regenerate}"
```

Comment:
The `host` and `port` shouldn't be statically set. It is user configurable during app start `khoj --host <host-ip> --port <port_number>`